exonum/src/messages/raw.rs

```diff
@@ -257,6 +262,19 @@ impl MessageWriter {
     }
 }
 
+// This is a trait that is required for technical reasons:
+// you can't make associated constants object-safe. This
```

Comment:
Interestingly enough, if I understand correctly, the problem is with `Transaction` rather than `Message`. Arguably, `Transaction` (at least in the form implemented by services) should not inherit from `Message` at all; the first represents the wire format, the second - business logic.